crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -571,17 +606,14 @@ impl<'a> Visitor<'a> for BodyVisitor<'a> {
     }
 }
 
-fn extract_raised_exception<'a>(
-    semantic: &SemanticModel<'a>,
-    exc: &'a Expr,
-) -> Option<QualifiedName<'a>> {
-    if let Some(qualified_name) = semantic.resolve_qualified_name(exc) {
-        return Some(qualified_name);
-    }
-    if let Expr::Call(ast::ExprCall { func, .. }) = exc {
-        return extract_raised_exception(semantic, func.as_ref());
-    }
-    None
```

Comment:
I got rid of this helper by using `map_callable()` before passing the node to `resolve_qualified_name` in `visit_stmt`